doc/user_guide/interactions.rst

```diff
@@ -55,7 +55,7 @@ Here is a simple scatter-plot created from the ``cars`` dataset:
         color='Origin:N'
     )
 
-We can create a variable parameter using :func:`parameter`, and assign that parameter a default value of 0.1 using the ``value`` property, as follows:
+We can create a variable parameter using :func:`alt.param`, and assign that parameter a default value of 0.1 using the ``value`` property, as follows:
```

Comment:
I can see that there is a mix of `alt.param` and `parameter` used currently on this page. I think the approach most consistent with the rest of the docs would be to change all these to just say `param`. Could you update all the occurrences of this on the page?